rust/arrow/src/util/pretty.rs

```diff
@@ -27,18 +27,18 @@ use prettytable::{Cell, Row, Table};
 use crate::error::{ArrowError, Result};
 
 ///! Create a visual representation of record batches
-pub fn pretty_format_batches(results: &Vec<RecordBatch>) -> Result<String> {
```

Comment:
Something to point out: one thing I noticed that thought was a good change was to make this API. Technically should have been done on the last PR (#6972) but I didn't want to hold that PR up any longer.

Reply:
I agree. I got into a bad habit of using &Vec instead of slice early on and have been meaning to start fixing this.